tests/package-title-dir-rtl-001/META-INF/container.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8" ?>

```

Comment:
I think this directory name does not match the proposed naming in https://github.com/w3c/epub-tests#naming (which is open for discussion).

Based on that, I think the directory name should be confreq-rs-pkg-dir_rtl (and similar for the other tests).

If you agree, please update the test ID to match, of course.

Reply:
I have changed to `confreq-rs-pkg-dir_rtl-001` (and -002, 003, etc.). But it is not really clear cut. Some of the tests are related to the interplay between `dir=rtl` and `dir=ltr` or `dir=auto`, ie, one could also consider to use `confreq-rs-pkg-dir-auto-001` (there is a separate `confreq-rs-pkg-dir-auto` anchor in the spec). But I presume we will have such border cases elsewhere, so the naming should be a bit flexible and, at the end of the day, the best judgement should prevail...

Reply:
Done it in https://github.com/w3c/epub-tests/pull/31/commits/dc9f0d396bd4d814812b3d707225bef4f0f59357

Reply:
Sounds good to me.